### PR TITLE
fixed batch jobs on Postgres

### PR DESCRIPTION
**Pull request by StevenXLi**
- changed warning message columns from LOB to varchar, and limit the size to 4000(max for varchar columns)
- changed how new warnings are append to the field, limiting the string to 4000 length, favouring new warnings

### CODE REVIEW

**Review by lukedegruchy — APPROVED** [review]

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java [hunk 1]
```diff
@@ -277,6 +277,18 @@ protected void init680() {
 			.unique(false)
 			.withColumns("CONCEPT_MAP_GRP_ELM_PID")
 			.onlyAppliesToPlatforms(NON_AUTOMATIC_FK_INDEX_PLATFORMS);
+
```
[the remark this answers is not in the document]
Comment (StevenXLi) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java:280]: I already changed that to drop the column and re-add it back with the right type

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java [hunk 2]
```diff
@@ -277,6 +277,27 @@ protected void init680() {
...
+		// have to drop the column first, orcacle doesn't play well with converting clobs to varchar
```
Comment (michaelabuckley) [inline hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java:281]: Can we mark the earlier migrations as .noop(), please?

**Review by michaelabuckley — APPROVED** [review]
Minor request to mark the earlier migrations as noop.  You might have to also mark the later drop as failok.

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java (hunk 2 above)
Comment (StevenXLi) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java]: done

**Pull request merged**